client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -65,20 +60,15 @@ function ReaderSubscriptionListItem( {
 				/>
 			</div>
 			<div className="reader-subscription-list-item__byline">
-				<span className="reader-subscription-list-item__site-title">{ <a href={ streamUrl }> { siteTitle } </a> }</span>
+				<span className="reader-subscription-list-item__site-title">
+					{ <a href={ streamUrl } className="reader-subscription-list-item__link"> { siteTitle } </a> }
+				</span>
+				<div>{ siteExcerpt }</div>
 				{ ! isEmpty( authorName ) &&
-					<span>
-						<span className="reader-subscription-list-item__by-text">
-							{ translate( 'by' ) }
-						</span>
-						<span><a href={ streamUrl }> { authorName } </a></span>
+					<span className="reader-subscription-list-item__by-text">
```

Comment:
I did this but it threw an error: `Use localize( ReactComponent ) instead of this.translate`?